plugins/nf-schema/src/main/nextflow/validation/validators/JsonSchemaValidator.groovy

```diff
@@ -28,24 +32,41 @@ import nextflow.validation.validators.evaluators.CustomEvaluatorFactory
 @Slf4j
 public class JsonSchemaValidator {
 
-    private ValidatorFactory validator
-    private Pattern uriPattern = Pattern.compile('^#/(\\d*)?/?(.*)$')
+    // TODO Change the base URL once the meta schema has been officially released
+    final private Map<String,String> supportedCustomParameterDrafts = [
+        "https://github.com/nextflow-io/schema-spec/raw/refs/heads/main/parameters_meta_schema.json": "/schemas/parameters/1.0/parameters_meta_schema.json"
```

Comment:
This has to be versioned surely? `main` will be a moving target.

Reply:
Yes that's what the `TODO` statement is for :). I can't put a versioned URL there as it doesn't exist yet